fossevents/static/css/main.scss

```diff
@@ -1,3 +1,54 @@
 @import 'layout';
 @import 'ui';
 @import 'events_detail';
+
+body {
+background-color: rgb(220, 220, 220);
+height:100%;
+}
+#container {
+min-height:100%;
+position:relative;
+overflow:auto;
+
+}
+.inner{
+background:#e3e3e3;
+margin-bottom:70px;
+}
+.inner1{
+
+margin-bottom:70px;
+}
+
+a {
+    text-decoration: none !important;
+}
+.row{
+padding-top:10px;
+}
+span{
+font-size: 13pt;
+font-weight: normal;
+margin: 0px;
+}
+.verticalLine {
+border-left: thick solid #ff0000;
+}
+.panel-heading>ul
+{
+list-style-type: none;
+}
+.panel-body{
+font-weight:normal;
+}
+#footer {
+position:absolute;
+bottom:0px;
+width:100%;
+padding:0px 10px  0px 10px;/* Height of the footer */
+background:#e6e6e6;
+min-height:100%;
+position:relative;
+overflow:auto;
+}
```

Comment:
Please see how existing css is written and organised. Ideally, you don't want to write any css rule inside `main.scss`, you should be able to decide, which rule belongs to which category. Create new file/category when needed, e.g. for all the global typography related rules, you can create a `_typography.scss` file and import that file in `main.scss` by writing `@import "typography";`